src/types.jl

```diff
@@ -37,7 +37,7 @@ Base.show(io::IO, s::WSymbol) = print(io, 'W', '"', s.name, '"')
 Base.:(==)(a::WSymbol, b::WSymbol) = a.name == b.name
 
 macro W_str(str)
-    WSymbol(str)
+    quote WSymbol($(esc(Meta.parse("\"$(escape_string(str))\"")))) end
```

Comment:
This is wrong: `W"name"` should just return a symbol

Reply:
It does.
```julia
julia> import Pkg; Pkg.activate("."); using MathLink
  Activating project at `~/dev/sw/MathLink.jl`

julia> let e=W`Sin[Cos[$(log(2))]]`
           println(e)
           println(weval(e))
       end
W"Sin"(W"Cos"(0.6931471805599453))
0.6955886362231636

julia> let e=W"name"
           println(e)
           println(typeof(e))
           println(weval(W"Head"(e)))
       end
name
MathLink.WSymbol
Symbol
```

Reply:
Ah, got it.